community/kernel/src/main/java/org/neo4j/kernel/impl/api/CommandVisitor.java

```diff
@@ -41,60 +39,29 @@
 import org.neo4j.kernel.impl.transaction.command.Command.RelationshipGroupCommand;
 import org.neo4j.kernel.impl.transaction.command.Command.RelationshipTypeTokenCommand;
 import org.neo4j.kernel.impl.transaction.command.Command.SchemaRuleCommand;
-import org.neo4j.kernel.impl.transaction.log.CommandWriter;
 
 /**
- * A CommandHandler has to handle all type of commands that a Neo4j transaction can generate. It provides
- * methods for handling each type of {@link Command} available through a callback pattern to avoid dynamic dispatching.
- * Implementations need to provide all these methods of course, but it is expected that they will delegate
- * the actual work to implementations that hold related functionality together, using a Facade pattern.
- * For example, it is conceivable that a {@link CommandWriter} would use a {@link CommandHandler}.
- * <p>
- * A CommandHandler must also be capable of visiting commands for a batch of transactions, the access pattern
- * goes like this:
- * <ol>
- * <li>{@link #begin(TransactionToApply)} called for the transaction to now visit commands for</li>
- * <li>...one or more commands for that transaction</li>
- * <li>{@link #end()}</li>
- * <li>{@link #begin(TransactionToApply)} called for the next transaction in the same batch</li>
- * <li>...one or more commands for that transaction</li>
- * <li>{@link #end()}</li>
- * <li>...same as above for every transaction in this batch</li>
- * <li>{@link #apply()}</li>
- * <li>{@link #close()}</li>
- * </ol>
- * The boolean returned from visit methods is false for continuing traversal, and true for breaking traversal.
+ * An interface for dealing with commands, either reading or writing them. See also {@link TransactionApplier}. The
+ * methods in this class should almost always return false, unless something went wrong.
```

Comment:
I did not find the place where we return true from `#visitX` methods. Could you please point me to it?